src/peft/__init__.py

```diff
@@ -83,5 +83,6 @@
     set_peft_model_state_dict,
     shift_tokens_right,
     load_peft_weights,
+    cast_non_trainable_to_dtype,
```

Comment:
We could update the section that I just recently added to the docs to use this function now, right?

https://github.com/huggingface/peft/blob/c6b28a22b86b2547cbf4443a9348a9e232f85c9e/docs/source/developer_guides/troubleshooting.md#L52-L54

Reply:
Done.